torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1959,13 +1980,13 @@ def print_results(grouped, indent, result_fn):
         if self._dynamic_results:
             grouped_dynamic_results = group(self._dynamic_results, args_index)
             buf += "\nThe following dimensions CAN be dynamic."
-            buf += "\nYou can use the following code to specify the constraints they must satisfy:"
+            buf += "\nPlease use the following code to specify the constraints they must satisfy:"
             buf += f"\n```\ndef specify_constraints{str(signature)}:"
             buf += f"\n{indent}return ["
             print_results(
                 grouped_dynamic_results,
                 indent * 2,
-                lambda result: f"{remove_default_lower_bound(result)},",
```

Comment:
Should `remove_default_lower_bound` be deleted then?

Reply:
OOC, why is `remove_default_lower_bound` not called any more?

Reply:
Moved its functionality upstream.